docs/tutorial/cpp/src/memory_tile.md

```diff
@@ -1,44 +1,42 @@
+
 # Memory Tile
 
-With the skeleton in place, let's look at the first element of the game, the memory tile. It will be the
-visual building block that consists of an underlying filled rectangle background, the icon image. Later we'll add a
-covering rectangle that acts as a curtain. The background rectangle is declared to be 64 logical pixels wide and tall,
-and it's filled with a soothing tone of blue. Note how lengths in the `.slint` language have a unit, here
-the `px` suffix. That makes the code easier to read and the compiler can detect when your are accidentally
-mixing values with different units attached to them.
+With the skeleton code in place, this step looks at the first element of the game, the memory tile. It's the
+visual building block that consists of an underlying filled rectangle background, the icon image. Later steps add a covering rectangle that acts as a curtain.
+
+Declare the background rectangle as 64 logical pixels wide and tall
+filled with a soothing tone of blue.
 
-We copy the following code into the `memory.slint` file:
+Note how lengths in Slint have a unit, here, the `px` suffix.
+This makes the code easier to read and the compiler can detect when you accidentally
+mix values with different units attached to them.
+
+Copy the following code into `ui/appwindow.slint` file, replacing the current content:
 
 ```slint
 {{#include memory_tile.slint:main_window}}
 ```
 
-Note that we export the <span class="hljs-title">MainWindow</span> component. This is necessary so that we can later access it
-from our business logic.
+The code exports the <span class="hljs-title">MainWindow</span> component. This is necessary so that the C++ code can access it later for business logic.
 
-Inside the <span class="hljs-built_in">Rectangle</span> we place an <span class="hljs-built_in">Image</span> element that
-loads an icon with the <span class="hljs-built_in">@image-url()</span> macro. The path is relative to the folder in which
-the `memory.slint` is located. This icon and others we're going to use later need to be installed first. You can download a
-[Zip archive](https://slint.dev/blog/memory-game-tutorial/icons.zip) that we have prepared.
+Inside the <span class="hljs-built_in">Rectangle</span> place an <span class="hljs-built_in">Image</span> element that
+loads an icon with the <span class="hljs-built_in">@image-url()</span> macro. The path is relative to the location of `ui/appwindow.slint`.
 
-If you are on Linux or macOS, download and extract it with the following two commands:
+You need to install this icon and others you use later first. You can download a pre-prepared
+[Zip archive](https://slint.dev/blog/memory-game-tutorial/icons.zip) to the `ui` folder and extract it with the
+following commands:
 
 ```sh
+cd ui
 curl -O https://slint.dev/blog/memory-game-tutorial/icons.zip
 unzip icons.zip
+cd ..
 ```
 
-If you are on Windows, use the following commands:
```

Comment:
I'm curious, why did you remove the Windows specific instructions?

Reply:
@tronical I think my reasoning was for a bit of consistency, as we don't tell people to build every time at each step, but I did leave the non-Windows commands in. What do you think? Do we need to tell people to build after every step, or would a C++ programmer know that?

Reply:
Hm, the instructions here aren't about building, they're part of the initial setup phase for fetching the icons package. Those differ quite a bit between Windows and macOS/Linux.

For building the instructions are always the same IIRC, just the way of launching is different - and you kept that difference.

So it's just these setup instructions where I'm curious why you remove the Windows specific bits (powershell usage).

Reply:
@tronical Purely and oversight, brought them back

Reply:
Thanks :)